gazebo_plugins/worlds/gazebo_ros_diff_drive_demo.world

```diff
@@ -14,11 +14,15 @@
 
   Try listening to TF:
 
-    ros2 run tf2_ros tf2_echo odom chassis
+    ros2 run tf2_ros tf2_echo odom_demo chassis
 
-    ros2 run tf2_ros tf2_echo chassis right_wheel
+    ros2 run tf2_ros tf2_echo chassis right_wheel0
 
-    ros2 run tf2_ros tf2_echo chassis left_wheel
+    ros2 run tf2_ros tf2_echo chassis left_wheel0
+
+    ros2 run tf2_ros tf2_echo chassis right_wheel1
+
+    ros2 run tf2_ros tf2_echo chassis left_wheel1
```

Comment:
How about leaving the diff drive demo as it is, and adding a separate demo for skid steer? Most people will use just a diff drive setup with 2 wheels, so it would be nice if the demo were as simple as possible. Also, people looking for skid steer would find it more easily if it were a separate demo.